spread.yaml

```diff
@@ -0,0 +1,89 @@
+project: snapcraft-rocks
+
+# note: the `path` is inside /root because the docker snap has issues
+# mounting directories that are outside the user's home
+path: /root/snapcraft-rocks
+environment:
+  PROJECT_PATH: /root/snapcraft-rocks
+  SNAPD_TESTING_TOOLS: $PROJECT_PATH/tools/external/tools
+  PATH: /snap/bin:$PATH:$SNAPD_TESTING_TOOLS:$PROJECT_PATH/tools/spread
+  # These variables are used to set the base in multi-base projects
+  SNAPCRAFT_BASE: core22
+  SNAPCRAFT_BUILD_BASE: core22
+
+include:
+    - tests/
+    - tools/
+
+backends:
+  google:
+    key: '$(HOST: echo "$SPREAD_GOOGLE_KEY")'
+    location: snapd-spread/us-east1-b
+    halt-timeout: 2h
+    systems:
+      - ubuntu-20.04-64:
+          workers: 2
+          storage: 40G
+      - ubuntu-22.04-64:
+          workers: 2
+          storage: 40G
+      - fedora-37-64:
```

Comment:
oh thanks a lot for catching this! I copy-pasted from rockcraft but there's nothing stopping this from using fedora 39. Updated.
Unfortunately the docker version pin still needs to be in place because even Fedora 39 doesn't have a recent enough snapd